pkg/apis/management.cattle.io/v3/cluster_types.go

```diff
@@ -74,6 +74,7 @@ const (
 	ClusterDriverRKE      = "rancherKubernetesEngine"
 	ClusterDriverK3s      = "k3s"
 	ClusterDriverK3os     = "k3os"
+	ClusterDriverRke2     = "rke2"
```

Comment:
nit: `ClusterDriverRKE2`. Not a big deal, and/or can be done later.

Reply:
used it the detection process, should mark rke2 clusters.